lablib/renderers/slate_render.py

```diff
@@ -73,11 +130,12 @@ def render(self) -> None:
                 f"""'{timecode.replace('"', "")}'""",
```

Comment:
jop the timecode issue i noted is only spotted now since spending some love on the slates.
played around a bit and for me this simple refactor actually did the trick lol.

```suggestion
                timecode,
```

Reply:
I can confirm that this is fixing the issue.